mjpc/tasks/humanoid/gait/task.xml

```diff
@@ -0,0 +1,93 @@
+<mujoco model="Humanoid Locomotion">
+  <include file="../../common.xml"/>
+  <include file="../humanoid.xml" />
+  <size memory="400K"/>
+  
+  <custom>
+    <text name="task_transition" data="Humanoid|Gait|Get Up|Flip|Hand Stand" />
```

Comment:
Flip and handstand work? 

I would expect the shoulders would need to be modified for handstand?

Reply:
These behaviors are prototyped and work with slowdown, although not with 100% reliability. 
A modified humanoid `humanoid_strong.xml` increases joint limits and gearing to achieve the behaviors. However, the dynamics become more unstable. We should probably transition to the CMU humanoid for these tasks.